docs/release/trg-5/trg-5-2.md

```diff
@@ -5,6 +5,7 @@ title: TRG 5.02 - Chart structure
 | Author               | Status | Created     | Post-History    |
 |----------------------|--------|-------------|-----------------|
 | Catena-X System Team | Active | 10-Nov-2022 | Initial release |
+| Catena-X System Team | Active | 10-Jan-2023 | Content updated |
```

Comment:
Imagine you read documentation and come to this history table. Is it informative or usefull? I would expect the history to be more precise, e.g. "add prerequisite and TL;DR". This would contain a hint what has changed.